src/lib/EventListener/ContentProxyCreateDraftListener.php

```diff
@@ -77,6 +77,8 @@ public function create(ContentProxyCreateEvent $event): void
             []
         );
 
+        $options->set('content_draft', $contentDraft);
```

Comment:
Can we Introduce a constant, so you can use it in taxonomy and notice the connection between this class and yours?